rotkehlchen/data_import/importers/binance.py

```diff
@@ -63,9 +72,11 @@ class BinanceSingleEntry(BinanceEntry, metaclass=abc.ABCMeta):
     """
     AVAILABLE_OPERATIONS: tuple[str, ...]
 
-    def is_entry(self, requested_operation: str) -> bool:
+    def is_entry(self, requested_operation: str, account: str, change: AssetAmount) -> bool:  # pylint: disable=unused-argument
         """This method checks whether row with "requested_operation" could be processed
-        by a class on which this method has been called
+        by a class on which this method has been called.
+        Some subclasses require combined checks with the "account" and "change" to
```

Comment:
nitpick: Here you should explain that account and change are keys of the CSV. Need to be explaining what arguments are.